build_packages

```diff
@@ -292,6 +293,25 @@ export KBUILD_BUILD_HOST="${BUILD_HOST:-pony-truck.infra.kinvolk.io}"
 info "Merging board packages now"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" "$@"
 
+info "Merging sysext packages now"
+for sysext in "${EXTRA_SYSEXTS[@]}"; do
+  IFS=":" read SYSEXT_NAME PACKAGE_ATOMS USEFLAGS < <(echo "$sysext");
+
+  info "Building packages for $SYSEXT_NAME sysext with USE=$USEFLAGS"
+  IFS=, read -ra atoms <<< "$PACKAGE_ATOMS"
+  for package in "${atoms[@]}"
+  do
+     USE="$USEFLAGS" FEATURES="-ebuild-locks binpkg-multi-instance" \
+       sudo -E "${EMERGE_CMD[@]}" \
```

Comment:
If you don't need `MODULES_SIGN_*` here...
```suggestion
     sudo env USE="$USEFLAGS" FEATURES="-ebuild-locks binpkg-multi-instance" \
       "${EMERGE_CMD[@]}" \
```

Reply:
We actually need the `MODULES_SIGN_*` here, as we are building it for the first time. We don't need it when installing for a second time from the binpkg.

Reply:
Passing through the whole environment is not ideal, but I just found out you can do `sudo --preserve-env=MODULES_SIGN_KEY,MODULES_SIGN_CERT`, which is slightly more convenient than `sudo env MODULES_SIGN_KEY="${MODULES_SIGN_KEY}" MODULES_SIGN_CERT="${MODULES_SIGN_CERT}"`.